desc/coils.py

```diff
@@ -716,21 +722,29 @@ class CoilSet(OptimizableCollection, _Coil, MutableSequence):
     Parameters
     ----------
     coils : Coil or array-like of Coils
-        collection of coils. Must all be the same type and resolution.
-    currents : float or array-like of float
-        currents in each coil, or a single current shared by all coils in the set
+        Collection of coils. Must all be the same type and resolution.
+    currents : float or array-like of float (optional)
+        Currents in each coil, or a single current shared by all coils in the set.
+    NFP : int (optional)
```

Comment:
Should include some more explanation of what it means if NFP>1 or sym=True. IE, that the set should only include the unique coils and they will be duplicated etc.